mne/beamformer/_lcmv.py

```diff
@@ -268,6 +325,8 @@ def make_lcmv(info, forward, data_cov, reg=0.05, noise_cov=None, label=None,
             continue
         Ck = np.dot(Wk, Gk)
 
+        # XXX This should be de-duplicated with DICS
```

Comment:
I guess we should also open an issue associated to this one, or refer to here in an existing issue.